src/VSCodeExtension/src/formatter/rules/control-structure.ts

```diff
@@ -0,0 +1,17 @@
+import { FormatRule } from "../formatter";
+
+/**
+ * Laves just one space after an if statement
+ *
+ * @param code input string
+ *
+ * @example `if                              (1 == 1){H(qs[0]);}` ->
+ * `if (1 == 1){H(qs[0]);}`
+ *
+ * more examples in the unit tests
+ */
+export const spaceAfterIf: FormatRule = (code: string): string => {
+    return code.replace(/(^| +|;|})if[ \n]*/, "$1if ");
```

Comment:
Using the `*` operator could make the regex match on zero instances of `[ \n]`, such that variable names starting with `if` could be affected. Do you want `+` here to match [1, ∞) copies of `[ \n]`?
```suggestion
    return code.replace(/(^| +|;|})if[ \n]+/, "$1if ");
```

Reply:
We also want to match none in the case of `if(...)` -> `if (...)`. I would suggest the following
```suggestion
    return code.replace(/(^| +|;|})if[ |\n]*\(/, "$1if ");
```

cc: @amrondonp who wrote this

Reply:
Good suggestion, @theRoughCode thanks!